adafruit_sgp40/__init__.py

```diff
@@ -31,7 +31,8 @@
 """
 from time import sleep
 from struct import unpack_from
-import adafruit_bus_device.i2c_device as i2c_device
+from adafruit_bus_device import i2c_device
+from adafruit_sgp40.voc_algorithm import VOCAlgorithm
```

Comment:
may want to have this only be imported if the voc algorithm is requested, since its a large py file